app-backup/borgbackup/borgbackup-1.2.0.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="Deduplicating backup program with compression and authenticated encryption"
+DESCRIPTION="BorgBackup (short: Borg) is a deduplicating backup program.
+Optionally, it supports compression and authenticated encryption.
+
+The main goal of Borg is to provide an efficient and secure way to backup data.
+The data deduplication technique used makes Borg suitable for daily backups
+since only changes are stored.
+The authenticated encryption technique makes it suitable for backups to not
+fully trusted targets."
+HOMEPAGE="https://www.borgbackup.org/"
+COPYRIGHT="2015-2022 The Borg Collective
+	2010-2014 Jonas Borgström"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/borgbackup/borg/archive/refs/tags/$portVersion.tar.gz"
```

Comment:
Updated per suggestions (left in Jonas name as in the LICENSE file), if not needed I can remove it
removed 2 unneeded requirements

Reply:
if space is not limited, it is good to have the full copyright, including jonas.

he didn't contribute to borg after we forked it, but he made all the stuff borg is based on in attic.

Reply:
I guess he did the groundwork, so best to mention him here